frontend/src/pages/JobList.tsx

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright 2021 Arrikto Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import CustomTable, { Column, Row, CustomRendererProps } from '../components/CustomTable';
+import RunUtils, { ExperimentInfo } from '../../src/lib/RunUtils';
+import { Apis, JobSortKeys, ListRequest } from '../lib/Apis';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { RoutePage, RouteParams } from '../components/Router';
+import { commonCss, color } from '../Css';
+import { formatDateString, errorToMessage } from '../lib/Utils';
+import Tooltip from '@material-ui/core/Tooltip';
+import { ApiJob, ApiTrigger } from '../apis/job';
+
+interface DisplayJob {
+  experiment?: ExperimentInfo;
+  job: ApiJob;
+  error?: string;
+}
+
+// Both masks cannot be provided together.
+type MaskProps = Exclude<
+  { experimentIdMask?: string; namespaceMask?: string },
+  { experimentIdMask: string; namespaceMask: string }
+>;
+
+export type JobListProps = MaskProps &
+  RouteComponentProps & {
+    disablePaging?: boolean;
+    disableSelection?: boolean;
+    disableSorting?: boolean;
+    hideExperimentColumn?: boolean;
+    noFilterBox?: boolean;
+    onError: (message: string, error: Error) => void;
+    onSelectionChange?: (selectedRunIds: string[]) => void;
+    jobIdListMask?: string[];
+    selectedIds?: string[];
+    refreshCount: number;
+  };
+
+interface JobListState {
+  jobs: DisplayJob[];
+}
+
+class JobList extends React.PureComponent<JobListProps, JobListState> {
+  private _tableRef = React.createRef<CustomTable>();
+
+  constructor(props: any) {
+    super(props);
+
+    this.state = {
+      jobs: [],
+    };
+  }
+
+  componentDidUpdate(prevProps: { refreshCount: number }) {
+    if (prevProps.refreshCount === this.props.refreshCount) {
+      return;
+    }
+    this.refresh();
+  }
+
+  public render(): JSX.Element {
+    const columns: Column[] = [
+      {
+        customRenderer: this._nameCustomRenderer,
+        flex: 1.5,
+        label: 'Job Name',
+        sortKey: JobSortKeys.NAME,
+      },
+      { customRenderer: this._statusCustomRenderer, label: 'Status', flex: 0.5 },
+      { customRenderer: this._triggerCustomRenderer, label: 'Trigger', flex: 1 },
+      { label: 'Start time', flex: 1, sortKey: JobSortKeys.CREATED_AT },
```

Comment:
I just kept the same notation as in the `RunList` page: https://github.com/kubeflow/pipelines/blob/master/frontend/src/pages/RunList.tsx#L111

Reply:
And I think it makes sense, in the code we have a `CREATED_AT` key which basically corresponds to a start time, and "Start time" sounds better :)

Reply:
I understand that the `Start time` makes sense in the `RunList` page, because a run was started after it is created. My understanding on `job` is a little bit different, `job` is a **configuration** which can trigger job at designated time and frequency. A similar concept is `cronjob config file`. The main difference is that `job` can be enabled and disabled. When a job is disabled, `Start time` might cause confusion. Do you think it makes sense?

Reply:
Also for `RunList`, there are two different fields for `Created at` and `Started at`: 
![Screen Shot 2021-02-17 at 11 29 16 AM](https://user-images.githubusercontent.com/37026441/108257382-8c792900-7113-11eb-99f4-2e7b34a51da3.png)

So the most accurate usage is to use `startedAt`, or use `creationTimestamp` with column name `Created at`: https://github.com/kubeflow/pipelines/blob/HEAD/frontend/src/pages/RunDetails.tsx#L892-L908

Reply:
I agree with zijianjoy on this, recurring runs are created at that time, not started.